Makefile.frag

```diff
@@ -1,34 +1,45 @@
 
-INSTALL_DIR=$(shell pwd)/build/install
+BUILD_DIR=$(shell pwd)/build
+DEPS_DIR=$(BUILD_DIR)/deps
+INT_DIR=$(BUILD_DIR)/install
```

Comment:
What INT_DIR means? Is INSTALL_STAGING ?

Reply:
INT_DIR = intermediate dir, where things are built to, but not necessarily their final destination